src/fluree/db/json_ld/policy.cljc

```diff
@@ -488,10 +489,12 @@
          (throw e))))))
 
 
-(defn policy-opts
-  [opts]
-  (-> (select-keys opts [:did :role :credential])
-      not-empty))
+(defn policy-identity
+  [{:keys [context] :as identity-map}]
+  (when-let [{:keys [role] :as identity} (-> (select-keys identity-map [:did :role :credential])
```

Comment:
Just a very nitpick-y style thing you can take or leave, but the threading macro might be doing its job* even better if `identity-map` alone were the first arg.

* Making nested forms more readable for humans

Reply:
agreed

Reply:
For some reason I'm not following what the suggestion is, what should the change be?

Reply:
Oh wait, I think I get it, you mean on its own as the first arg to `->`